updatecli/scripts/install-third-party.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+
+# Check if the base directory exists
+if [ ! -d "$REPOSITORY_DIR" ]; then
+    echo "Error: Base directory $base_directory does not exist."
```

Comment:
where does `$base_directory` come from?

Reply:
+1 here
what happen if $base_directory is unset?

Since you are using bash, you could leverage parameter substitution as explained on 
https://tldp.org/LDP/abs/html/parameter-substitution.html

I usually like to start my bash script with 

```
set -eux
```

Reply:
Ops, copy and paste problems. Sorry. Let me fix it .